recipes/nanodbc/all/conanfile.py

```diff
@@ -62,7 +62,7 @@ def requirements(self):
         if self.options.with_boost:
             self.requires("boost/1.73.0")
```

Comment:
Any reason to keep the outdated boost?

Reply:
no idea, but probably out of the scope of this PR

Reply:
I didn't notice.  I can add it to this PR

Reply:
Too late :)